src/main/resources/public-editor-api.yaml

```diff
@@ -31,6 +40,92 @@ paths:
           description: User with this email already exists.
         400:
           description: Required parameters are missing or invalid.
+  /user/{userId}:
+    get:
+      description: Get the user details given his userId.
+      summary: getUsersByUserid
+      operationId: GetUsersByUserId
```

Comment:
Should be singular i.e. GetUserByUserId if we're only returning a unique user (matching a unique ID)